platform/vs/docker-sonic-vs/start.sh

```diff
@@ -27,17 +27,18 @@ popd
 
 [ -d /etc/sonic ] || mkdir -p /etc/sonic
 
-if [[ -f /usr/share/sonic/virtual_chassis/default_config.json ]]; then
-    CHASS_CFG="-j /usr/share/sonic/virtual_chassis/default_config.json"
-fi
-
 # Note: libswsscommon requires a dabase_config file in /var/run/redis/sonic-db/
 # Prepare this file before any dependent application, such as sonic-cfggen
 mkdir -p /var/run/redis/sonic-db
 cp /etc/default/sonic-db/database_config.json /var/run/redis/sonic-db/
 
 SYSTEM_MAC_ADDRESS=$(ip link show eth0 | grep ether | awk '{print $2}')
-sonic-cfggen -t /usr/share/sonic/templates/init_cfg.json.j2 -a "{\"system_mac\": \"$SYSTEM_MAC_ADDRESS\"}" $CHASS_CFG > /etc/sonic/init_cfg.json
+sonic-cfggen -t /usr/share/sonic/templates/init_cfg.json.j2 -a "{\"system_mac\": \"$SYSTEM_MAC_ADDRESS\"}" > /etc/sonic/init_cfg.json
+
+if [[ -f /usr/share/sonic/virtual_chassis/default_config.json ]]; then
+    sonic-cfggen -j /etc/sonic/init_cfg.json -j /usr/share/sonic/virtual_chassis/default_config.json --print-data > /tmp/init_cfg.json
```

Comment:
why dont we write directly to /etc/sonic/init_cfg.json

Reply:
The template /usr/share/sonic/templates/init_cfg.json.j2 does not have j2 code to take the /usr/share/sonic/virtual_chassis/default_config.json contents as data and generate the result with the default_config,json contents in /etc/sonic/init_cfg.json

Reply:
ok, but do you need to write to /tmp/init_cfg.json or can you update init_cfg.json?

Reply:
I think it is possible to update the init_cfg.json directly since sonic-cfggen loads all the contents first as data and then produces the output). But for here, I just followed the model used for generating config_db.json (given just below these lines from 43-45) to appear consistent

Reply:
@vganesan-nokia Does this mean that "-t" & "-j" does not work togather in sonic-cfggen?

```
-j JSON, --json JSON  json file that contains additional variables
```

```
-t TEMPLATE, --template TEMPLATE
                        render the data with the template file
```

Reply:
Yes. When -t option is used the template file, the contents of the json file given with option -j are loaded as data to the template. If the template does not have explicit j2 code to take these data, the rendering will not have the json files contents in the output.

Reply:
Thanks for details, now I understand.